### PR TITLE
setup_foreman_discovery: fix template unlocking

### LINKED ISSUE #409: setup_foreman_discovery: fix template unlocking
```
run: hammer -u admin -p changeme template update --name "PXELinux global default" --lock "false"
out: Could not update the config template:
out:   Error: Unrecognised option '--lock'
```
it should be `--locked`

### PR DESCRIPTION
**Pull request by lpramuk**
setup_foreman_discovery: fix template unlocking
- fixed typo s/lock/locked/

(Fixes #409)

### CODE REVIEW

File: automation_tools/__init__.py [hunk 1]
```diff
@@ -630,8 +630,8 @@ def setup_foreman_discovery(sat_version, upstream=False):
     for daemon in ('foreman', 'httpd', 'foreman-proxy'):
             manage_daemon('restart', daemon)
     # Unlock the default Locked template for discovery
-    run('hammer -u admin -p {0} template update --name '
-        '"PXELinux global default" --lock "false"'
+    run('hammer -u admin -p {0} template update '
+        '--name "PXELinux global default" --locked "false"'
```
Comment (sghai) [inline automation_tools/__init__.py]: :+1:

**Comment by sghai** [comment]
ACK

File: automation_tools/__init__.py [hunk 2]
```diff
@@ -630,8 +630,8 @@ def setup_foreman_discovery(sat_version, upstream=False):
     for daemon in ('foreman', 'httpd', 'foreman-proxy'):
             manage_daemon('restart', daemon)
     # Unlock the default Locked template for discovery
-    run('hammer -u admin -p {0} template update --name '
-        '"PXELinux global default" --lock "false"'
+    run('hammer -u admin -p {0} template update '
```
Comment (jyejare) [inline automation_tools/__init__.py]: @lpramuk , You can make use of hammer function from module instead of fabric run.

**Comment by jyejare** [comment]
ACK with non blocker comment.

**Pull request merged**